runnc.go

```diff
@@ -319,13 +321,44 @@ func main() {
 
 	Config.RuncBinaryPath = DefaultRuncBinaryPath
 
+	logPath := ""
+
+	for i, v := range args {
+		if v == "--log" {
```

Comment:
What if the args look like: `--log --abc`? wouldn't `--abc` be confusd as the logPath?

Reply:
yup. This is how docker calls it, so it's a temporary hack till we replace the parsing to https://github.com/urfave/cli

Reply:
Got it, thanks. Can you add a quick check so that `args[i+1]` doesn't start with `--` please (that might help a little bit)?

Reply:
Added additional check!

Reply:
Thanks. Shouldn't it be 'HasPrefix'?

Reply:
Fixed.